frontend/css/year-in-music.less

```
@@ -568,7 +568,8 @@
       overflow: hidden;
       .graph {
         color: var(--selectedColor);
-        overflow: auto;
+        overflow: scroll;
```

I think this should allow  scrolling horizontally when needed (mobile sizes), while not showing the scrollbar when unnecessary:
```suggestion
        overflow-x: auto;
```

---

Done!